pipe/errored_datums_reader.yaml

```diff
@@ -0,0 +1,67 @@
+---
+pipeline:
+  name: errored_datums_reader
+transform:
+  image_pull_secrets:
+  - battelleecology-quay-read-all-pull-secret
+  image: quay.io/battelleecology/errored_datums_reader:test5
+  cmd:
+  - /bin/bash
+  stdin:
+  - '#!/bin/bash'
+  - python3 -m errored_datums_reader.main
+  env:
+    LOG_LEVEL: DEBUG
+  secrets:
+  - name: pdr-secret
+    envVar: DB_HOST
+    key: hostname
+  - name: pdr-secret
+    envVar: DB_USER
+    key: username
+  - name: pdr-secret
+    envVar: DB_PASSWORD
+    key: password
+  - name: pdr-secret
+    envVar: DB_NAME
+    key: database
+  - name: pdr-secret
+    envVar: DB_SCHEMA
+    key: schema
+  - name: errored-datums-repo-reader-secret
+    envVar: AUTHORIZATION_TOKEN
+    key: token
+input:
+  cron:
+    name: tick
+    spec: "@never"
+    # overwrite: true
+autoscaling: true
+scheduling_spec:
+  node_selector:
+    cloud.google.com/gke-ephemeral-storage-local-ssd: "true"
+    nodepool.neonscience.org/pipeline: "yes"
+    cloud.google.com/gke-spot: "true"
+pod_spec: |-
+  { "tolerations": [
+    {
+      "key": "nodepool.neonscience.org/pipeline",
+      "operator": "Exists"
+    },
+    {
+      "effect": "NoSchedule",
+      "key": "cloud.google.com/gke-spot",
+      "operator": "Exists"
+    }  
+  ] }
+pod_patch: |-
```

Comment:
FYI there are dedicated user and storage resource requests & limits now available for the pipeline spec. See any li191r pipeline for examples.